src/IDBStore.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var IDBStore = {
-  indexedDB: function() {
+  indexedDB: () => {
```

Comment:
Method syntax here would be even more compact. Would Emscripten's various parsers support it?

Reply:
I have confirmed if closure compiler can lower ES6 classes to ES6 without runtime support functions (that is currently the bar for including ES6 feature):  See https://github.com/emscripten-core/emscripten/blob/fc0f28c46a16970e7cdc7bcd4ce4f7f831a9c90b/test/test_other.py#L12599.   My guess is that its  not possible.. at least at this point in time.

Reply:
Actually you are correct.. method syntax seems to work fine, enough ourside of a JS class!  Thanks for the pointer.  I will try to land that change first.